classes/class.tapestry-node.php

```diff
@@ -147,6 +151,9 @@ public function set($node)
         if (isset($node->behaviour) && is_string($node->behaviour)) {
             $this->behaviour = $node->behaviour;
         }
+        if (isset($node->mediaDuration) && is_numeric($node->mediaDuration)) {
+            $this->mediaDuration = $node->mediaDuration;
+        }
```

Comment:
I think it needed to be moved up to avoid overwriting changes from some other code that also set `mediaDuration`. However, I removed that code later, so it no longer makes a difference.

For consistency's sake, I have moved this block of code back.